strapi/strapi
# fix: fix for type of relation icons in the relation modal

**What does it do?**

we have relation in collection types we have content and those content have relations in the basic settings as one to many, many to one, those relation icons we are getting as rectangle, so fixed that css now we can get the relation of types in square.

**Why is it needed?**

To fix issue [21350](https://github.com/strapi/strapi/issues/21350 ): Type of relation icons aren’t squares but rectangle in the relation modal

**How to test it?**

- go to CTB
- select any one content which have relation
- click on edit in basic settings we can see relations with rectangle

**Related issue(s)/PR(s)**

fixes [21350](https://github.com/strapi/strapi/issues/21350)
